charactersheet/charactersheet/models/common/armor.js

```diff
@@ -74,7 +74,7 @@ function Armor() {
     });
 
     self.armorWeightLabel = ko.pureComputed(function() {
-        return self.armorWeight() ? self.armorWeight() + ' lbs.' : '';
+        return self.armorWeight() !== '' && self.armorWeight() >= 0 ? self.armorWeight() + ' lbs.' : '';
```

Comment:
Instead of an empty string, can we return `0 lbs.` instead? Makes the UI look better.